test/test_yaml_utils.py

```diff
@@ -202,8 +203,7 @@ def test_custom_ruamel_yaml_emitter(
     assert output == expected_output
 
 
-@pytest.fixture(name="yaml_formatting_fixtures")
-def fixture_yaml_formatting_fixtures(fixture_filename: str) -> tuple[str, str, str]:
```

Comment:
I made this a fixture so that any issues here would show up as a test error instead of a test failure. Why did you change it to a helper function?

Reply:
It made the testing harder and also maintenance, vscode got bit confused by it. I also think that having a test failure is fine.